libs/partners/google-vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -9,6 +9,7 @@
 from typing import Any, Dict, Iterator, List, Optional, Union, cast
 from urllib.parse import urlparse
 
+import proto  # type: ignore
```

Comment:
whats type: ignore for?

Reply:
this is the error i got. 
langchain_google_vertexai/chat_models.py:12:1: error: Skipping analyzing "proto": module is installed, but
missing library stubs or py.typed marker  [import-untyped]
    import proto

Reply:
@baskaryan  I update the code base on your suggestion. Please let me know if you have other concern. Thank you!